data/darktableconfig.xml.in

```diff
@@ -1222,6 +1222,13 @@
     <shortdescription>scroll to darkroom modules when expanded/collapsed</shortdescription>
     <longdescription>when this option is enabled then darktable will try to scroll the module to the top of the visible list</longdescription>
   </dtconfig>
+  <dtconfig prefs="gui" section="miscellaneous">
```

Comment:
this is no "miscellaneous" section, just remove the section here, just letting prefs attribute.

Reply:
Yeah, cheers.  Just updating now.